sig-multicluster/charter.md

```diff
@@ -0,0 +1,16 @@
+# SIG Multicluster Charter
+
+This charter is a WIP.
+
+SIG Multicluster focuses on solving common challenges related to:
+
+- Managing/administering multiple Kubernetes clusters
```

Comment:
Probably rewording needed.
`Managing/administering centralised information about multiple kubernetes clusters` ?? or something else.
The prior (`cluster administration/management` as the term) seems to suggest stuff `cluster-lifecycle` folks might be involved with.
I understand that the same wording is currently used on the sig-multicluster page; IMHO needs correction.

Reply:
I think there's probably an overlap here. As I understand it, cluster-lifecycle is about creating new clusters. As I understand it, the wording currently in the README pertains to administering multiple clusters from the same source of truth. I'm fine striking it for now if people would like, but I expect this to be in scope for SIG multicluster.

Reply:
Seems reasonable, lets go ahead with what this PR has.